docs/data/charts/legend/LegendPositionNoSnap.js

```diff
@@ -43,59 +45,59 @@ export default function LegendPositionNoSnap() {
           defaultValue: 'middle',
           options: ['left', 'middle', 'right'],
         },
-        {
-          propName: 'padding',
-          knob: 'number',
-          defaultValue: 0,
-        },
         {
           propName: 'itemsNumber',
           knob: 'number',
-          defaultValue: 5,
+          defaultValue: 3,
           min: 1,
           max: data.length,
         },
       ]}
-      renderDemo={(props) => (
+      renderDemo={(
+        /** @type {{ itemsNumber: number | undefined; direction: "horizontal" | "vertical";  vertical: "top" | "middle" | "bottom"; horizontal: "left" | "middle" | "right"; }} */
+        props,
+      ) => (
         <PieChart
           series={[
             {
               data: data.slice(0, props.itemsNumber),
             },
           ]}
+          height={400}
+          margin={{
+            top: 20,
+            bottom: 20,
+            left: 40,
+            right: 40,
+          }}
```

Comment:
Why not the same value for every direction?

The pie chart might look better with a smaller value than 400.

The 400 was taking into account the space for margin (100 in each direction) to get same size pie chart, it should be `height={240}` and `width={280}` with your current margin